go.mod

```diff
@@ -259,6 +258,7 @@ require (
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/pierrec/lz4 v2.6.1+incompatible // indirect
 	github.com/pjbgf/sha1cd v0.2.3 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
Seems the library is still used indirectly by other external packages. Not sure if we can remove this dep completely.

Reply:
I understand indirect references, meaning that other packages we reference reference this package, but kyverno does not reference this package